### DISCUSSION TITLE
Alguien para colaborar en un proyecto

### DISCUSSION THREAD

**Discussion by Dfredude** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#40] [General]
Hola, seria buena idea poder contribuir en algun proyecto real en equipo para poder ir aplicando todo lo aprendido en el bootcamp y darle utilidad. **_Busco alguien que en el desarrollo del proyecto quiera trabajar el Front-End con Angular y Back-End con Java Springboot_** de preferencia. Saludos

Contacto: dominguezlucio@outlook.com

**Comment by YairRuizMx** [comment]
Suena interesante! Yo me ofrezco como Tester! Jaja

**Reply by Dfredude** [reply]
Jaja pasame tu correo para mandarte una invitacion al servidor de Discord!

**Comment by JLuisPrz** [comment]
Me gusta la idea, también quiero contribuir.

**Comment by iononi** [comment]
Estoy interesado, podría apoyar en el backend.

**Comment by luisced** [comment]
Yo también podría aportar en el front

**Comment by Jimdrer** [comment]
Me apunto como Front End, Ux/Ui

**Comment by JoseAngelSil** [comment]
me gusta la idea con todo gusto

**Reply by Dfredude** [reply]
Perfecto! Pasame tu correo o algo para mandarte el link del grupo en privado.

**Comment by SHNAITHED** [comment]
Me agrada la idea!! cuenta conmigo!!

**Reply by Dfredude** [reply]
Perfecto! Pasame tu correo o algo para mandarte el link del grupo en privado.

**Comment by RafBD** [comment]
Hola, me gustaría apoyar en el front :)

**Reply by Dfredude** [reply]
Perfecto! Pasame tu correo o algo para mandarte el link del grupo en privado.

**Reply by RafBD** [reply]
Si, mi correo es rafa.bediaz@gmail.com

**Comment by SciDev98** [comment]
Estaria interesante ayudar en ello . estoy abierto a ello.

**Comment by serapioMartinez** [comment]
Suena intereasante, me gustaria apoyar en donde se necesite del proyecto

**Reply by RupertoTacuche** [reply]
Yo tambien estoy interesado

**Comment by RupertoTacuche** [comment]
Yo tambien estoy interesado. Soy ing en mecatronica y he trabajado con sistemas embebidos e internet de las cosas.
